tests/client-dbus/tests/dbus/pool/test_create_filesystem.py

```diff
@@ -102,21 +103,21 @@ class CreateFSTestCase1(SimTestCase):
     """
 
     _POOLNAME = "deadpool"
-    _VOLNAME = "thunk"
+    _FSNAME = "thunk"
 
     def setUp(self):
         """
         Start the stratisd daemon with the simulator.
         """
         super().setUp()
         self._proxy = get_object(TOP_OBJECT)
-        self._devs = _DEVICE_STRATEGY()
+        devs = _DEVICE_STRATEGY()
         ((_, (poolpath, _)), _, _) = Manager.Methods.CreatePool(
             self._proxy,
-            {"name": self._POOLNAME, "redundancy": (True, 0), "devices": self._devs},
+            {"name": self._POOLNAME, "redundancy": (True, 0), "devices": devs},
```

Comment:
Given that this is local, do the same thing as [here](https://github.com/stratis-storage/stratisd/pull/1710/files#diff-1de2f36d829f7f87ccc8647b0450d353R84).